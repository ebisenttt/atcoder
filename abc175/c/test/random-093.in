-447860709433155 473153971399094 257283053744950
