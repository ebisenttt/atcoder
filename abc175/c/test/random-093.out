66705398056745
